src/node.js

```diff
@@ -92,6 +92,22 @@
         process.argv[1] = path.resolve(process.argv[1]);
 
         var Module = NativeModule.require('module');
+
+        // check if user passed `-c` or `--check` arguments to Node.
+        if (process._syntax_check_only != null) {
+          var vm = NativeModule.require('vm');
+          var fs = NativeModule.require('fs');
```

Comment:
this could go either way cause it's not at all consistent in the file, we might be shifting that way but there's only one use of `const` amongst the 20 `NativeModule.require` calls already in this file, someone's probably going to do a round of cleanup at some point and this change could be done then